diff --git a/steam.desktopo b/steam.desktop
index 77314e0..be7ab21 100644
--- a/steam.desktopo
+++ b/steam.desktop
@@ -27,7 +27,7 @@ Comment[th]=โปรแกรมสำหรับจัดการและ
 Comment[tr]=Steam üzerinden oyun oynama ve düzenleme uygulaması
 Comment[uk]=Програма для керування іграми та запуску ігор у Steam
 Comment[vi]=Ứng dụng để quản lý và chơi trò chơi trên Steam
-Exec=/usr/bin/steam %U
+Exec=/usr/bin/steam -no-cef-sandbox %U
 Icon=steam
 Terminal=false
 Type=Application
